corehq/apps/app_manager/static/app_manager/js/summary/models.js

```diff
@@ -124,6 +124,17 @@ hqDefine('app_manager/js/summary/models', function() {
                 formUrl: formData.form_url,
             });
         };
+        self.moduleReference = function(moduleId) {
```

Comment:
Optional nit: This could be DRYed up a little with `moduleFormReference`, couldn't it?